salt/modules/lxd.py

```diff
@@ -3213,8 +3213,44 @@ def image_alias_delete(image,
 #####################
 
 
-def snapshots_all(container=None, remote_addr=None,
-                  cert=None, key=None, verify_cert=True):
+def snapshots_all(container, remote_addr=None, cert=None, key=None, verify_cert=True):
+    '''
+    Get all snapshots for a container
+
+    container :
+        The name of the container to get.
+
+    remote_addr :
+        An URL to a remote Server, you also have to give cert and key if
+        you provide remote_addr and its a TCP Address!
+
+        Examples:
+            https://myserver.lan:8443
+            /var/lib/mysocket.sock
+
+    cert :
+        PEM Formatted SSL Certificate.
+
+        Examples:
+            ~/.config/lxc/client.crt
+
+    key :
+        PEM Formatted SSL Key.
+
+        Examples:
+            ~/.config/lxc/client.key
+
+    verify_cert : True
+        Wherever to verify the cert, this is by default True
```

Comment:
I don't know what `Wherever to verify the cert...` means in this context. Could we make this more clear, please?